src/core/public/core_app/status/components/metric_tiles.tsx

```diff
@@ -46,3 +121,10 @@ const formatMetric = ({ value, type }: Metric) => {
 const formatMetricId = ({ name }: Metric) => {
   return name.toLowerCase().replace(/[ ]+/g, '-');
 };
+
+const formatDelayFooterTitle = (values: number[], type?: DataType) => {
```

Comment:
Having this formatting in-line in the component makes it harder to read. We could eventually extract all the formatting helpers into their own file if we end up adding a lot of them.